core/user/_user/userProps.js

```diff
@@ -0,0 +1,14 @@
+import * as R from 'ramda'
+
+export const keysProps = {
+  props: 'props',
```

Comment:
'props' is not a key of the user props... you can use ObjectUtils.keys.props to refer to it